wgpu/src/lib.rs

```diff
@@ -1465,6 +1466,12 @@ pub struct VertexState<'a> {
     /// The name of the entry point in the compiled shader. There must be a function with this name
     /// in the shader.
     pub entry_point: &'a str,
+    /// Specifies the values of pipeline-overridable constants in the shader module module.
+    ///
+    /// The key represents the numeric ID of the constant, if one is specified, and otherwise the constant's identifier name.
+    ///
+    /// The value may represent any of WGSL's concrete scalar types.
```

Comment:
question: I doubt it, but: is it worth linking to `Scalar` docs here?

This question applies to other places where this doc has been copy-pasted.

Reply:
idk really, maybe we can clarify it once someone runs into this?

Reply:
I consider this conversation resolved, but I'll leave it up in case others might be interested in this question. Feel free to `Resolve`, if you want.